### DISCUSSION TITLE
Downloading maps for specific area

### DISCUSSION THREAD

**Discussion by habitoti** [osmandapp/OsmAnd#14355] [Maps Related]
Maybe I am doing this completely wrong, but hopefully someone can give me hint: when looking at the map in "new areas" a message comes up allowing me to download the detailed maps for it. However when I somehow miss to confirm that right away, the message disappears and I seem to be unable to force it to come again. Also, I have no clue what area would actually be covered then. What I want to do is either frame a part of the map and have all maps downloaded that would cover that area, or at least pick a track and have all affected map areas downloaded. 
Yesterday I drove along a track where suddenly I entered an area where I had no detailed map data, and also wasn't keen on downloading it with mobile data. So I would really be able to prepare that at home when still on WiFi somehow. Especially ahead of vacations with maybe multiple tracks to cover, it would be great to have a straightforward procedure to make sure to have downloaded all affected data upfront.
How are you guys doing that?

**Comment by stonecrusher** [comment]
Usually I zoom out and then zoom in again and the prompt should reappear.

Covered area should at least include the official area that it is named after. I usually look it up on https://www.openstreetmap.org/ 
Search for the name and then click on "boundary" on the left.
example:
![2022-05-25 09_39_36-](https://user-images.githubusercontent.com/1388389/170208132-29ecc2db-9802-4e89-9f0a-b95c4391a07f.png)

I have no solution for preloading all related data to tracks. I'm just zooming around areas where I will be and download everything offered.

**Comment by habitoti** [comment]
Yeah, that's kind of how I do it now as well, but it feels a bit like a workaround. Would be great to have a function on tracks to simply download the affected areas explicitly.
